README.md

```diff
@@ -305,6 +305,11 @@ We use the [Natural Earth Data Cultural Vectors](http://www.naturalearthdata.com
 ## Contribute
 Want to help? Join us on slack at [http://slack.tmrow.co](http://slack.tmrow.co).
 
+### Logger
+
+We have a public [logger](https://kibana.electricitymap.org/app/kibana) which shows warnings and errors for
```

Comment:
Maybe we should give a direct link to https://kibana.electricitymap.org/app/kibana#/discover/1710fdd0-2460-11e8-a779-9d01de8d7a71